onnxscript/ir/serde.py

```diff
@@ -511,7 +511,7 @@ def deserialize_model(proto: onnx.ModelProto) -> _core.Model:
         model_version=_get_field(proto, "model_version"),
         doc_string=_get_field(proto, "doc_string"),
         functions=functions,
-        meta_data_props=deserialize_metadata_props(proto.metadata_props),
+        metadata_props=deserialize_metadata_props(proto.metadata_props),
```

Comment:
May be good to add a negative test as well: eg., two nodes which have identical inputs, but differ in one attribute value.

Reply:
You should also have a test checking that names in subgraphs are renamed (operator If, Scan, Loop).

Reply:
Added test_the_ops_with_the_same_inputs_but_different_attributes_are_not_csed